examples/realtime-quickstart-react/src/pages/ws/receive-message.tsx

```
@@ -0,0 +1,150 @@
+          });
+          break;
+        case WebsocketsEventType.CONVERSATION_MESSAGE_DELTA:
+        case WebsocketsEventType.CONVERSATION_MESSAGE_COMPLETED:
+          if (event.data.type === 'question') {
+            break;
+          }
+          // 处理语音文本消息
+          const content = event.data.content;
+          console.log('content', content);
+          setMessageList(prev => {
+            // 如果上一个事件是增量更新，则附加到最后一条消息
```

Biome (1.9.4)

[error] Other switch clauses can erroneously access this declaration.
Wrap the declaration in a block to restrict its access to the switch clause.

The declaration is defined in this switch clause:

Unsafe fix: Wrap the declaration in a block.


(lint/correctness/noSwitchDeclarations)